packages/atomic/src/components/common/atomic-timeframe/atomic-timeframe.stories.tsx

```diff
@@ -14,7 +14,7 @@ const {defaultModuleExport, exportedStory} = defaultStory(
         }
       </style>
     `,
-    parentElement: () => {
+    parentElement: function () {
```

Comment:
`parentElement() {` doesn't work, and `parentElement: () => {` doesn't work either. Looks like the transpiler tried to simplify something and messed up.